src/Microsoft.AspNet.Mvc.Core/project.json

```diff
@@ -5,8 +5,8 @@
         "warningsAsErrors": true
     },
     "dependencies": {
-        "Microsoft.AspNet.Cors.Core": "1.0.0-*",
         "Microsoft.AspNet.Hosting.Abstractions": "1.0.0-*",
+        "Microsoft.AspNet.Http": "1.0.0-*",
```

Comment:
Now the question becomes, "what's this for"?

Let's merge this one and take a look at that next.